stdlib/pervasives.gr

```diff
@@ -1,19 +1,26 @@
+/* grainc-flags --no-pervasives */
+
 /**
  * @module Pervasives: This module is automatically imported into every Grain program. You can think of it as the global environment. Although it is automatically imported, it can still be imported manually.
  *
- * @example import Pervasives from "pervasives"
+ * @example include "pervasives"
  *
  * @since v0.1.0
  */
 
-import Exception from "runtime/exception"
-import Memory from "runtime/unsafe/memory"
-import WasmI32 from "runtime/unsafe/wasmi32"
+module Pervasives
+
+include "runtime/exception"
+include "runtime/unsafe/memory"
+include "runtime/unsafe/wasmi32"
 
-import { equal as (==) } from "runtime/equal"
-import { compare } from "runtime/compare"
+include "runtime/equal"
+from Equal use { equal as (==) }
```

Comment:
Do we really want `runtime/equal` to be available as `Equal`? Why not `RuntimeEqual`?

Reply:
I'm with the convention that filename should be the same as the module name. That also works out if we expose the runtime as it's own package, since you'd reference it as `Runtime.Equal`.

Reply:
I think we need to rework our directory-structure-as-namespacing design here and actually do the `Runtime.Equal`, etc changes in a `runtime.gr` module that provides things to the rest of the stdlib because this is chaotic.

I also think that the convention could easily be PascalCaseOfTheFilepath: `runtime/equal` is `RuntimeEqual`, `runtime/unsafe/wasmi32` is `RuntimeUnsafeWasmI32` but it'd be better if we just didn't suck at naming and organizing in the first place.

Reply:
#1637